AK/Assertions.cpp

```diff
@@ -10,7 +10,7 @@
 #include <AK/StringBuilder.h>
 #include <AK/StringView.h>
 
-#if (defined(AK_OS_LINUX) && !defined(AK_OS_ANDROID)) || defined(AK_LIBC_GLIBC) || defined(AK_OS_BSD_GENERIC) || defined(AK_OS_SOLARIS) || defined(AK_OS_HAIKU)
+#if defined(AK_LIBC_GLIBC) || defined(AK_OS_ANDROID) || defined(AK_OS_BSD_GENERIC) || defined(AK_OS_SOLARIS) || defined(AK_OS_HAIKU)
```

Comment:
In `LibJS/JIT/NativeExecutable.cpp` I used `__has_include(<execinfo.h>)`. I wonder why we list a bunch of platforms here instead of using that.

Reply:
Should i force push with that change?

Reply:
@ADKaster you initially introduced this code. Was there a particular reason you didn't use `__has_include`?

Reply:
I forgot it existed 😅 that's definitely more elegant than a platform ifdef chain.